contracts/contracts/RequestManager.sol

```diff
@@ -138,6 +208,16 @@ contract RequestManager is Ownable {
         resolutionRegistry = ResolutionRegistry(_resolutionRegistry);
     }
 
+    /// Create a new transfer request.
+    ///
+    /// @param targetChainId ID of the target chain.
+    /// @param sourceTokenAddress Address of the token contract on the source chain.
+    /// @param targetTokenAddress Address of the token contract on the target chain.
+    /// @param targetAddress Recipient address on the target chain.
+    /// @param amount Amount of tokens to transfer. Does not include fees.
+    /// @param validityPeriod The number of seconds the request is to be considered valid.
+    ///                       A request cannot be fulfilled once its validity period has elapsed.
```

Comment:
That sounds a bit confusing. Technically the request can be filled anytime. But the request cannot be claimed anymore and the user is entitled to withdraw the tokens if there are not active claims.
I'm not sure how to describe this properly.

Reply:
> 



> That sounds a bit confusing. Technically the request can be filled anytime. But the request cannot be claimed anymore and the user is entitled to withdraw the tokens if there are not active claims. I'm not sure how to describe this properly.

This is a very good point. Yeah, it's a bit tricky to convey the meaning.
I went with
```
Once its validity period has elapsed, the request cannot be claimed
anymore and will eventually expire, allowing the request submitter 
to withdraw the deposited tokens if there are no active claims.    
```